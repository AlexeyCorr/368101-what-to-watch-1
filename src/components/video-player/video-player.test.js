import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

const mock =  {
  picture: `img/bohemian-rhapsody.jpg`,
  src: [`1.webm`, `2.mp4`],
  isPlaying: false
};

it(`renders correctly`, () => {
  const {picture, src, isPlaying} = mock;
  const tree = renderer
    .create(<VideoPlayer
      isPlaying={isPlaying}
      picture={picture}
      src={src}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
