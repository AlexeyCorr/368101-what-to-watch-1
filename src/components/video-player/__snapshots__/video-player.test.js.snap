// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<video
  className="player__video"
  muted={true}
  poster="img/bohemian-rhapsody.jpg"
>
  <source
    src="2.mp4"
    type="video/mp4"
  />
</video>
`;
